packages/ui/src/composites/wui-card-select-loader/styles.ts

```diff
@@ -5,7 +5,7 @@ export default css`
     display: flex;
     flex-direction: column;
     align-items: center;
-    width: 76px;
+    width: 104px;
```

Comment:
are you sure this doesn't break other views?

Reply:
Yeah we only use that specific component in all wallets view